src/collections/_documentation/platforms/php/symfony2.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Symfony2
+sidebar_order: 60
+---
+
+Monolog
+=======
+
+Capturing Errors
+----------------
+
+Monolog supports Sentry out of the box, so you'll just need to configure a handler:
+
+.. sourcecode:: php
+
+    $client = new \Sentry\Client('___PUBLIC_DSN___');
+
+    $handler = new Monolog\Handler\RavenHandler($client);
```

Comment:
This will not work with the client 2.0. We will need to add a `SentryHandler` to Monolog.